Tests/test_imagefont.py

```diff
@@ -660,6 +661,7 @@ def test_variation_get(self):
             {"name": b"Size", "minimum": 0, "maximum": 300, "default": 0}
         ]
 
+    @pytest.mark.skipif(is_mingw(), reason="epsilon too high for meaningful test")
```

Comment:
I'm not sure what is causing the difference, but I am getting epsilon around 33 for both variation tests. Checking manually, I see there is a significantly different kerning after the first letter, but the output looks correct to me.

Note: MSYS2 is currently using FreeType version 2.10.2.

Reply:
Please could you attach images for each case?

Reply:
Sure.

test_variation_set_by_name:
a.png (MSYS):
![a](https://user-images.githubusercontent.com/3819630/82825586-cf231300-9eab-11ea-9d29-b8e6d6168049.png)

b.png (target - variation_adobe.png):
![b](https://user-images.githubusercontent.com/3819630/82825597-d3e7c700-9eab-11ea-91bd-b6d20c9761e7.png)


test_variation_set_by_axes
a.png (MSYS):
![a](https://user-images.githubusercontent.com/3819630/82825721-02fe3880-9eac-11ea-891a-ba56a2637a42.png)

b.png (target - variation_adobe_axes.png):
![b](https://user-images.githubusercontent.com/3819630/82825770-1a3d2600-9eac-11ea-8fc8-48b706e55124.png)

Basic and Raqm layout are pixel identical. In fact, it looks like there is a bug in the tests where the used `layout_engine` is not being explicitly set and defaults to Raqm in both cases.